lib/Twig/Environment.php

```diff
@@ -242,13 +258,9 @@ public function setCache($cache)
      */
     public function getCacheFilename($name)
```

Comment:
should this method be deprecated too ? there is nothing forcing to use a filename as cache key (if the cache is not a filesystem one, it is very unlikely)